test/todoListController.spec.js

```diff
@@ -0,0 +1,67 @@
+describe('TodoListController', function() {
+  beforeEach(module('TodoList'));
+
+  var ctrl;
+
+  beforeEach(inject(function($controller) {
+    ctrl = $controller('TodoListController');
+  }));
+
+  it('initialises with an empty task list and empty add tasks box', function() {
+    // expect(ctrl.taskList).toBeUndefined();
+    expect(ctrl.newTask).toBeUndefined();
+  });
+
+  describe('when adding a task', function() {
+
+    it('displays the task in the to do list', function() {
+      ctrl.newTask = 'Learn Angular';
+      ctrl.addTask();
+      expect(ctrl.taskList).toEqual([{title:'Learn Angular', done:false}]);
+    });
+  });
+
+  describe('when adding another task', function() {
+
+    var items = [{title:'Learn Angular', done:false}, {title:'Buy irn bru', done:false}]
+
+    it('displays both the first task and the new task in the to do list', function() {
+      ctrl.newTask = 'Learn Angular';
+      ctrl.addTask();
+      ctrl.newTask = 'Buy irn bru'
+      ctrl.addTask();
+      expect(ctrl.taskList).toEqual(items);
+    });
+  });
+
+  describe('when marking a task done', function() {
+
+    xit('displays the task text in grey and struck through', function() {
+      ctrl.newTask = {title:'Find Nemo', done: false};
```

Comment:
Why not refactor the use of this mock data as you've done on line 26? You're typing it out on  lines 43 and 55 too.